DirectProgramming/C++SYCL_FPGA/Tutorials/Features/mem_channel/README.md

```diff
@@ -253,10 +244,10 @@ board.
      nmake fpga
      ```
 
-> **Note**: If you encounter any issues with long paths when compiling under
-Windows*, you may have to create your ‘build’ directory in a shorter path, for
-example c:\samples\build.  You can then run cmake from that directory, and
-provide cmake with the full path to your sample directory.
+>**Note**: If you encounter any issues with long paths when compiling under Windows*, you may have to create your 'build' directory in a shorter path, for example `C:\samples\build`. You can then run cmake from that directory, and provide cmake with the full path to your sample directory, for example
+> ```
+> C:\samples\build> cmake -G "NMake Makefiles" C:\long\path\to\code\sample\CMakeLists.txt
```

Comment:
I find this part overkill, don't you think?

Reply:
i always find an example helpful.

If I found it overkill i wouldn't have written it :)

Reply:
Yeah but here we are showing how to use cmake which is orthogonal to what the sample is trying to show.
We could also explain how to install cmake, etc. 
I feel like we should expect people to know how to use the tools (e.g. we expect people to know how to open a terminal, navigate to the appropriate folder, etc.) 
We give then the necessary info to run cmake in our particular context (the flags we use), but I don't view it as necessary to give them more general information such as how to run cmake from a different folder than the build directory that we just explained them above